### PR TITLE
Issue-1390 MongoDB Shutdown対応

### PR DESCRIPTION
**Pull request by hidekita-uhuru**
HerokuのMongoDBがShutdownされるため、MongoDBからPosgreSQLに乗り換え方針です。

### CODE REVIEW

File: settings.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ if (process.env.ISSUER) {
     process.env.MONGO_URI ||
```
Comment (edcop2) [inline settings.js:70]: MongoDB envs unnecessary

File: package.json [hunk 2]
```diff
@@ -21,8 +21,9 @@
     "express-session": "^1.11.3",
     "jsonwebtoken": "^5.0.4",
     "mongodb": "^2.2.24",
```
Comment (edcop2) [inline package.json:23]: MongoDB unnecessary

File: settings.js [hunk 3]
```diff
@@ -12,30 +12,30 @@ var settings = {
   functionGlobalContext: {}, // enables global context
   httpNodeCors: {
     origin: '*',
-    methods: 'GET,PUT,POST,DELETE'
+    methods: 'GET,PUT,POST,DELETE',
```
Comment (edcop2) [inline settings.js:15]: this is prettier error that adds unnecessary `,`.
While not major, can be easily fixed by adding `.prettierrc` copied from enebular

**Review by edcop2 — APPROVED** [review]

**Review by edcop2 — APPROVED** [review]

File: settings.js (hunk 1 above)
Comment (hidekita-uhuru) [reply settings.js:70]: Thank you, I removed it

File: package.json (hunk 2 above)
Comment (hidekita-uhuru) [reply package.json:23]: Thank you, I removed it.

**Review by junichikatsu — APPROVED** [review]

File: settings.js (hunk 3 above)
Comment (hidekita-uhuru) [reply settings.js:15]: I add .prettierrc file from enebular and save files again.

**Pull request merged**